lib/scripts.js

```diff
@@ -314,10 +314,10 @@ var scripts = {
     It checks if the job in the top of the delay set should be moved back to the
     top of the  wait queue (so that it will be processed as soon as possible)
   */
-  updateDelaySet: function(queue, delayedTimestamp) {
-    var keys = [queue.keys.delayed, queue.keys.active, queue.keys.wait];
 
-    var args = [queue.toKey(''), delayedTimestamp];
+  updateDelaySet: function(queue, delayedTimestamp){
+    var keys = [queue.keys.delayed, queue.keys.active, queue.keys.wait, queue.keys.priority];
+    var args = [queue.toKey(''), delayedTimestamp, queue.token];
```

Comment:
```queue.token``` does not seem to be needed.

Reply:
You're right. The token is useless there. I thought I'll emit the same waiting event like when adding a job, but then I realised it's not a good idea right now.

So I removed the corresponding line but forgot to remove the arguments.